kraken/kubernetes/client.py

```diff
@@ -274,15 +274,16 @@ def get_job_status(name, namespace="default"):
 
 
 # Obtain node status
-def get_node_status(node):
+def get_node_status(node, timeout=60):
     try:
-        node_info = cli.read_node_status(node, pretty=True)
+        node_info = cli.read_node_status(node, pretty=True, _request_timeout=timeout)
```

Comment:
The parameter must be named as _request_timeout in Kubernetes python client. Just want to make sure that's the case. Thoughts?

Reply:
Yes I think it is properly named 


```
 def read_node_status(self, name, **kwargs):  # noqa: E501
        """read_node_status  # noqa: E501

        read status of the specified Node  # noqa: E501
        This method makes a synchronous HTTP request by default. To make an
        asynchronous HTTP request, please pass async_req=True
        >>> thread = api.read_node_status(name, async_req=True)
        >>> result = thread.get()

        :param async_req bool: execute request asynchronously
        :param str name: name of the Node (required)
        :param str pretty: If 'true', then the output is pretty printed.
        :param _preload_content: if False, the urllib3.HTTPResponse object will
                                 be returned without reading/decoding response
                                 data. Default is True.
        :param _request_timeout: timeout setting for this request. If one
                                 number provided, it will be total request
                                 timeout. It can also be a pair (tuple) of
                                 (connection, read) timeouts.
        :return: V1Node
                 If the method is called asynchronously,
                 returns the request thread.
        """
```